<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changepasswd</class>
 <widget class="QDialog" name="changepasswd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>245</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>382</width>
    <height>245</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>382</width>
    <height>245</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>修改密码</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
    border-width: 1px;
    padding: 1px;
    border-style: solid;
	border-color: rgb(22, 102, 57);
    border-radius: 5px;
}

/* As mentioned above, eliminate the padding and increase the border. */
QLineEdit:focus:focus {
    border-width: 3px;
    padding: 0px;
}
#changepasswd{
background-color: rgb(235, 244, 234);
}
.QPushButton:!hover
{
background-color: rgb(162, 210, 152);
color: rgb(255, 255, 255);
font: 9pt &quot;微软雅黑&quot;;
border-color: rgb(255, 255, 255);
border-radius:3px;
 padding: 3px;
}
.QPushButton:hover
{
background-color: rgb(85, 170, 127);
color: rgb(255, 255, 255);
font: 9pt &quot;微软雅黑&quot;;
border-color: rgb(255, 255, 255);
border-radius:3px
}
.QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>251</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>原密码：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="oldlineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>新密码：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="newlineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="newlineEdit2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
